operations/mimir-mixin/dashboards/tenants.libsonnet

```diff
@@ -159,6 +159,67 @@ local filename = 'mimir-tenants.json';
       ),
     )
 
+    .addRow(
+      $.row('Ingestion requests')
```

Comment:
Would it make sense to rename that to `Distributor ingestion requests`?

The title `Ingestion requests` alone sounds similar to `Ingester requests`, which would be misleading.

Reply:
```suggestion
      $.row('Distributor ingestion requests')
```

Reply:
Yes, good point.